script/Arcueid/Day 5/05_00_ARC05_9.txt

```diff
@@ -26,8 +26,9 @@ My brain creaks as if it's being squeezed by my skull.
 [sha:3f34268b02bd8e5c73d145c6e9d07b37c27678f9]{
 -- Page 562, Offset 10113.
 -- 俺は２分後の結末を予測し、破裂しそうな心臓をかき<毟|むし>る。
-// Not sure where the 'I want to' is coming from here?
-Predicting the outcome of the next two minutes, I want to tear out my heart, which is already ready to burst.
+// The second clause here is quite strange and could do with a re-check.
```

Comment:
Seems like it can be '[heartrending / heart is wrenched](https://ejje.weblio.jp/content/%E6%8E%BB%E3%81%8D%E6%AF%9F%E3%82%8B)', so maybe this is 
> My heart, already fit to burst, is rent clean in two when I imagine what's going to happen in the next two minutes.

but it's kinda awkward, feels like you'd need more words in EN